ui/pages/confirmations/confirmation/templates/switch-ethereum-chain.test.js

```diff
@@ -84,6 +88,10 @@ describe('switch-ethereum-chain confirmation', () => {
           },
         ],
       },
+      confirmAlerts: {
```

Comment:
Is this additional state needed for the alert system hooks to work?

Could we instead make the alert selectors tolerant of a missing reducer to minimise the diff and remove the snapshot changes also?

Reply:
I fixed selector, snapshot changes are due to an additional provider `TemplateAlertContextProvider`